tag.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 readonly PROGNAME=$(basename "$0")
-readonly PROGDIR=$(readlink -m "$(dirname "$0")")
+readonly PROGDIR=$(pwd)
```

Comment:
I'm not super familiar with sym links. `readlink` didn't work as expected for me on Mac, so I just switched to using `pwd`. I'm not too sure why we need this at all, but let me know if I should not check this in / am missing something

Reply:
Might be some legacy stuff due to $GOPATH/src being a symlink when the repo is checked out in Travis or Circle CI (I recall something related to that) or similar. Getting current dir via `pwd` seems reasonable.